src/pages/interim/deriv/_love-trading.tsx

```diff
@@ -1,17 +1,38 @@
 import React from 'react'
 import { LoveTradingComponent } from '../_love-trading'
 import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
 
-const content = {
+type LeftCTASectionProps = {
```

Comment:
seems like these love trading types are the same, can we create a type file and import these from there?

Reply:
Yeah that make sense 👍🏾